src/lib/Dungeon.js

```diff
@@ -6,14 +6,15 @@ class AutomationDungeon
     static Settings = {
                           FeatureEnabled: "Dungeon-FightEnabled",
                           StopOnPokedex: "Dungeon-FightStopOnPokedex",
-                          BossRush: "Dungeon-BossRush",
-                          SkipChests: "Dungeon-SkipChests"
+                          AvoidEncounters: "Dungeon-AvoidEncounters",
```

Comment:
Should not have changed the setting name, as it makes the setting reset to default. Not a big issue, but try not to change this as it messes with people's default.

If this needs to be done anyways, set the default from the old value from the old name.

Reply:
I did it on purpose, since the meaning changed a bit from the previous behaviour